packages/assets-controllers/src/AssetsContractController.ts

```diff
@@ -25,7 +30,7 @@ import { ERC721Standard } from './Standards/NftStandards/ERC721/ERC721Standard';
  * @param chainId - ChainID of network
  * @returns Whether the current network supports token detection
  */
-export const SINGLE_CALL_BALANCES_ADDRESS_BY_CHAINID: Record<Hex, string> = {
+export const SINGLE_CALL_BALANCES_ADDRESS_BY_CHAINID = {
```

Comment:
Narrows type to const-asserted literal properties. See changelog "Changed" section for details.